python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -35,38 +40,52 @@ def predict(m, x):
     m.predict(x)
 
 
-def _build_fil_classifier(m, data, arg={}):
+def _build_fil_classifier(m, data, arg={}, tmpdir=None):
     """Setup function for FIL classification benchmarking"""
     from cuml.utils.import_utils import has_xgboost
     if has_xgboost():
         import xgboost as xgb
     else:
         raise ImportError("No XGBoost package found")
 
-    # use maximum 1e5 rows to train the model
-    train_size = min(data[0].shape[0], 100000)
-    dtrain = xgb.DMatrix(data[0][:train_size, :], label=data[1][:train_size])
+    if isinstance(data[0], (pd.DataFrame, pd.Series)):
```

Comment:
Seems like we should break this out to a `_training_data_to_numpy(X, y)` function since it would be reusable.

Reply:
Should add a small unit test for this as well as it's a discrete little chunk of functionality with a bunch of conditionals

Reply:
Solved. Also added a simple test

Reply:
@JohnZed , can you take a quick look? Sorry about replying late - I had some problems with `cudf` and `rmm` version last week and eventually solved those today